gm-decision-tree/decision-tree-service.yaml

```diff
@@ -104,6 +104,49 @@ paths:
               schema:
                 $ref: '#/components/schemas/Errors'
 
+  /journeys/{journey-uuid}/questions/{question-uuid}:
+    get:
+      tags:
+        - get-journey-question
+      operationId: get-journey-question
+      parameters:
+        - name: journey-uuid
+          in: path
+          description: Journey unique ID
+          required: true
+          schema:
+            type: string
+            format: uuid
+        - name: question-uuid
+          in: path
+          description: Journey question unique ID
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        200:
+         description: Array of question definitions
```

Comment:
Looks good - only question is whether the response from this would be a question outcome, rather than array of question definitions?? (don't know though - just querying)

Reply:
I wondered about that - but in this case I think this is probably more intuitive (perhaps?) - i.e. we are simply requesting a specific question (as opposed to submitting answers for evaluation to produce an outcome (which can be of varying types)... 🤔